### PR TITLE
cmake: pass root cache variables to child images

### PR DESCRIPTION
**Pull request by tejlmand**
When a ROOT cache variable is set by the user in the parent image then those cache variables should be passed to child images so that the child images will use the same root folders.

Jira: NCSDK-28257

### CODE REVIEW

**Review by shanthanordic — APPROVED** [review]

**Comment by shanthanordic** [comment]
NEed +1 from @kylebonnici  and @magnev  before merging

**Review by gmarull — APPROVED** [review]

**Review by magnev — APPROVED** [review]

**Pull request merged**
